app/client/aclui/AccessRules.ts

```diff
@@ -1178,8 +1178,8 @@ Useful for examples and templates, but not for sensitive data.`),
   },
   SchemaEdit: {
     name: t("Permission to edit document structure"),
-    description: t("Allow editors to edit structure (e.g. modify and delete tables, columns, " +
-        "layouts), and to write formulas, which give access to all data regardless of read restrictions."),
+    description: t("Allow editors to edit structure (e.g. modify and delete tables, columns, \
+      layouts), and to write formulas, which give access to all data regardless of read restrictions."),
```

Comment:
I see that this is indeed missing from Weblate, and found some issues in i18next-scanner for this (https://github.com/i18next/i18next-scanner/issues/35 and https://github.com/i18next/i18next-scanner/issues/227). Thank you for finding and fixing!

Should the whitespace at the start of the continuation lines be removed? Otherwise the extra whitespace becomes part of the string.

BTW, it looks like there are a few other such instances of concatenated strings; I found some more using this command `grep -rI '\bt(.*+$' ./app`.

Reply:
Thanks, it's a good point